FraihmworkHealthAndIntegrityApi.yaml

```
@@ -1241,7 +1241,22 @@ components:
           description: Same as TOTAL UPTIME % for now. Will update definition as we gain ability to inject modules for handling different types of components (like using additional properties defined for radars)
           type: number
           format: double
-        
+            
+    FaultCounters:
+      type: object
+      description: This data type contains counters for faults against a given component relative to various reference times.
+      properties:
+      
+        faultsLifetime:
+          description: Total count of faults that have occured on a component since the TIME OF REGISTRATION.
+          type: integer
+          format: int64
+          
+        faultsSinceStart:
+          description: Total count of faults that have occured on a component since TIME OF STARTUP
```

Fix spelling of "occurred" and add period at end for consistency
```suggestion
          description: Total count of faults that have occurred on a component since TIME OF STARTUP.
```